src/screens/RegisterEnsSheet.js

```diff
@@ -147,20 +173,20 @@ export default function RegisterEnsSheet() {
             </Inline>
           )}
           <SheetActionButtonRow>
-            {isSuccess && debouncedSearchQuery.length > 2 && (
+            {isSuccess && !isNameInvalid ? (
+              <SheetActionButton
+                color={colors.green}
+                label="Continue on 􀆊"
+                onPress={() => null}
+                size="big"
+                weight="heavy"
+              />
+            ) : (
               <>
-                {registration.isRegistered ? (
+                {debouncedSearchQuery.length > 2 && (
```

Comment:
Doesn't this mean that if a name is available, it'll still show the Clear button (in addition to Continue On)? I don't necessarily think that's bad but just wondering bc it differs from what I see in Figma.